### ISSUE TITLE
Setting height to auto with minHeight resulting to zero height in wysiwyg mode

### ISSUE THREAD

**Issue by JaniL** [nhn/tui.editor#1757]
Hey, this issue is a bit similar with #1749 that was reported recently. This also happens on 3.0.2. I'm using Chrome 92.0.4515.107 on macOS.

![Screenshot 2021-08-17 at 9 41 21](https://user-images.githubusercontent.com/60955/129676388-19b5f873-387b-4781-9842-865c09478635.png)

I want to customize the editor so that it resizes automatically if the height of the content exceeds the minimum height. This works fine if the initial edit type is markdown. However if the initial edit type is wysiwyg (or mode switch is not hidden and user switches to wysiwyg), the content section of the editor does not have any height at all. I can however edit the content of the editor which seems to go out of bounds.

Here's the react-editor example code. Initially we were using `@toast-ui/editor` directly, but while we were investigating this we switched to react-editor and encountered the same issue there:

```
const toolbarItems = [
    ['heading', 'bold', 'italic', 'strike', 'quote'],
    ['ul', 'ol', 'task', 'indent', 'outdent'],
    ['link', 'code', 'codeblock'],
]

return (
      <Editor
        initialEditType="wysiwyg"
        minHeight={"550"}
        height="auto"
        initialValue={initialContent}
        toolbarItems={toolbarItems}
        hideModeSwitch={true}
      />
)
```

**Comment by JaniL** [comment]
And FWIW, this does not happen on 3.0.1.

**Comment by js87zz** [comment]
@JaniL 
Similar issues have been fixed in v3.0.3, so can you check it out?

**Comment by JaniL** [comment]
Ah yeah, so it seems! I was looking into this yesterday and left writing this issue for today, but did not realize that 3.0.3 came out. Thank you!
